includes/validation/class-amp-css-length.php

```diff
@@ -0,0 +1,136 @@
+<?php
+/**
+ * Class AMP_CSS_Length
+ *
+ * @package AMP
+ */
+
+/**
+ * Class AMP_CSS_Length
+ *
```

Comment:
Add a reference and link to the JS class that this was adapted from, and at which version.
As we copy behavior from the JS validator, we'll need to make sure we keep it in sync over time.